common/protobuf/Control.proto

```diff
@@ -0,0 +1,33 @@
+package Packet;
+
+message Control
+{
+    enum ShootMode
+    {
+        KICK = 1;
+        CHIP = 2;
+    }
+
+    enum TriggerMode
+    {
+        IMMEDIATE = 1;
+        ON_BREAK_BEAM = 2;
+        STAND_DOWN = 3;
+    }
+
+    enum Song
+    {
+        STOP = 1;
+        FIGHT_SONG = 2;
+    }
+
+    required float xVelocity = 1         [default = 0];
+    required float yVelocity = 2         [default = 0];
+    required float aVelocity = 3         [default = 0];
+    required float dVelocity = 4         [default = 0];
```

Comment:
:+1: sounds good, I missed that.  Maybe just put a comment for avelocity and dvelocity just to clarify what they are.